addons/web/static/tests/control_panel/custom_filter_item_tests.js

```diff
@@ -70,6 +70,35 @@ odoo.define('web.filter_menu_generator_tests', function (require) {
             cfi.destroy();
         });
 
+        QUnit.test('custom OR filter presets new condition from preceding', async function (assert) {
+            assert.expect(1);
+
+            const searchModel = new ActionModel();
+            const cfi = await createComponent(CustomFilterItem, {
+                props: {
+                    fields: this.fields,
+                },
+                env: { searchModel },
+            });
+
+            // Add first filter condition
+            await cpHelpers.toggleAddCustomFilter(cfi);
+            await testUtils.fields.editSelect(cfi.el.querySelector('.o_generator_menu_field'), 'color');
```

Comment:
maybe add two sanity checks here to ensure that 'color' and '!=' aren't the first value in their select